### ISSUE TITLE
SELinux violations for pmdakvm on debugfs and max_mmu_page_hash_collisions

### ISSUE THREAD

**Issue by martinpitt** [cockpit-project/bots#688]
Downstream report: https://bugzilla.redhat.com/show_bug.cgi?id=1817361

This is some packaging bug during image creation:
```
  Running scriptlet: pcp-selinux-5.0.3-2.fc32.x86_64                    449/539 
Re-declaration of type ipa_custodia_t
Failed to create node
Bad type declaration at /var/lib/selinux/targeted/tmp/modules/100/ipa_custodia/cil:1
semodule:  Failed!
```

Possible root cause: https://bugzilla.redhat.com/show_bug.cgi?id=1817968
May be fixed with https://bodhi.fedoraproject.org/updates/FEDORA-2020-94d7720a9d

**Comment by cockpituous** [comment]
verify/fedora-32
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testEnforcingToggle (__main__.TestSelinux)

DevTools listening on ws://127.0.0.1:9822/devtools/browser/fa12eb3a-0303-4f11-8218-08e2987ed560
[0326/134559.839506:ERROR:egl_util.cc(60)] Failed to load GLES library: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: cannot open shared object file: No such file or directory
[0326/134559.841477:ERROR:viz_main_impl.cc(180)] Exiting GPU process due to errors during initialization
[0326/134559.844512:WARNING:gpu_process_host.cc(1213)] The GPU process has crashed 1 time(s)
CDP: {"source":"security","level":"error","text":"Refused to apply inline style because it violates the following Content Security Policy directive: \"default-src 'self' http://127.0.0.2:9791\". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.\n","timestamp":1585230369797.212,"url":"http://127.0.0.2:9791/cockpit/$efcdacfc19afef159a38f4ef667def3a243b28d96df293a7ff6f1cd52c449134/shell/index.js","lineNumber":14,"stackTrace":"(minified)"}
> warning: grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error

audit: type=1400 audit(1585230380.908:323): avc:  denied  { read } for  pid=2357 comm="pmdakvm" name="kvm" dev="debugfs" ino=22735 scontext=system_u:system_r:pcp_pmcd_t:s0 tcontext=system_u:object_r:debugfs_t:s0 tclass=dir permissive=1
audit: type=1400 audit(1585230380.919:324): avc:  denied  { read } for  pid=2357 comm="pmdakvm" name="max_mmu_page_hash_collisions" dev="debugfs" ino=22773 scontext=system_u:system_r:pcp_pmcd_t:s0 tcontext=system_u:object_r:debugfs_t:s0 tclass=file permissive=1
Wrote JS log to TestSelinux-testEnforcingToggle-fedora-32-127.0.0.2-2901-FAIL.js.log
Journal extracted to TestSelinux-testEnforcingToggle-fedora-32-127.0.0.2-2901-FAIL.log.gz
Journal extracted to TestSelinux-testEnforcingToggle-fedora-31-127.0.0.2-2902-FAIL.log.gz
Traceback (most recent call last):
  File "/build/cockpit-project/bots/make-checkout-workdir/test/common/testlib.py", line 878, in tearDown
    self.check_journal_messages()
  File "/build/cockpit-project/bots/make-checkout-workdir/test/common/testlib.py", line 1072, in check_journal_messages
    raise Error("FAIL: Test completed, but found unexpected journal messages:\n" + first)
testlib.Error: FAIL: Test completed, but found unexpected journal messages:
audit: type=1400 audit(1585230380.908:323): avc:  denied  { read } for  pid=2357 comm="pmdakvm" name="kvm" dev="debugfs" ino=22735 scontext=system_u:system_r:pcp_pmcd_t:s0 tcontext=system_u:object_r:debugfs_t:s0 tclass=dir permissive=1

# Result testEnforcingToggle (__main__.TestSelinux) failed
# 1 TEST FAILED [51s on 1-ci-srv-04]
not ok 260 test/verify/check-setroubleshoot TestSelinux.testEnforcingToggle
```
First occurrence: 2020-03-26T13:46:27.623558 | revision df0c36c7cdc9f4037ad67a44ef862c0a763b3172
Times recorded: 1
Latest occurrences:

- 2020-03-26T13:46:27.623558 | revision df0c36c7cdc9f4037ad67a44ef862c0a763b3172

**Comment by martinpitt** [comment]
Fixed in refresh PR #741
